Documentation import incorrect `diesel::expression`

On the page: https://docs.rs/diesel/1.4.5/diesel/pg/expression/dsl/fn.any.html
It notes: 
> You can import it specifically from `diesel::expression::any`, or glob import `diesel::dsl::*`

I tried using the `diesel::expression::any` but this does not exist.
```
error[E0432]: unresolved import `diesel::expression::any`
 --> main.rs:8:5
  |
8 | use diesel::expression::any;
  |     ^^^^^^^^^^^^^^^^^^^^^^^ no `any` in `expression`

error: aborting due to previous error
```
`use diesel::pg::expression::dsl::any;` does work.

So maybe the documentation has to be updated here.

--- a/diesel/src/pg/expression/array_comparison.rs
+++ b/diesel/src/pg/expression/array_comparison.rs
@@ -9,7 +9,7 @@ use crate::sql_types::Array;
 ///
 /// As with most bare functions, this is not exported by default. You can import
 /// it specifically from `diesel::pg::expression::dsl::any`, or glob import
-/// `diesel::dsl::*`.
+/// `diesel::dsl::any`.
 ///
 /// # Example
 ///

--- a/diesel/src/pg/expression/array_comparison.rs
+++ b/diesel/src/pg/expression/array_comparison.rs
@@ -8,7 +8,7 @@ use crate::sql_types::Array;
 /// Creates a PostgreSQL `ANY` expression.
 ///
 /// As with most bare functions, this is not exported by default. You can import
-/// it specifically from `diesel::expression::any`, or glob import
+/// it specifically from `diesel::pg::expression::dsl::any`, or glob import
 /// `diesel::dsl::*`
 ///
 /// # Example
@@ -37,7 +37,8 @@ where
 /// Creates a PostgreSQL `ALL` expression.
 ///
 /// As with most bare functions, this is not exported by default. You can import
-/// it specifically as `diesel::dsl::all`.
+/// it specifically as `diesel::pg::expression::dsl::all`, or glob import
+/// `diesel::dsl::*`.
 ///
 /// # Example
 ///

--- a/diesel/src/pg/expression/array_comparison.rs
+++ b/diesel/src/pg/expression/array_comparison.rs
@@ -38,7 +38,7 @@ where
 ///
 /// As with most bare functions, this is not exported by default. You can import
 /// it specifically as `diesel::pg::expression::dsl::all`, or glob import
-/// `diesel::dsl::*`.
+/// `diesel::dsl::all`.
 ///
 /// # Example
 ///

--- a/diesel/src/pg/expression/array_comparison.rs
+++ b/diesel/src/pg/expression/array_comparison.rs
@@ -8,8 +8,7 @@ use crate::sql_types::Array;
 /// Creates a PostgreSQL `ANY` expression.
 ///
 /// As with most bare functions, this is not exported by default. You can import
-/// it specifically from `diesel::pg::expression::dsl::any`, or glob import
-/// `diesel::dsl::any`.
+/// it specifically from `diesel::pg::expression::dsl::any`, or `diesel::dsl::any`.
 ///
 /// # Example
 ///
@@ -37,8 +36,7 @@ where
 /// Creates a PostgreSQL `ALL` expression.
 ///
 /// As with most bare functions, this is not exported by default. You can import
-/// it specifically as `diesel::pg::expression::dsl::all`, or glob import
-/// `diesel::dsl::all`.
+/// it specifically as `diesel::pg::expression::dsl::all`, or `diesel::dsl::all`.
 ///
 /// # Example
 ///

--- a/diesel/src/pg/expression/array_comparison.rs
+++ b/diesel/src/pg/expression/array_comparison.rs
@@ -9,7 +9,7 @@ use crate::sql_types::Array;
 ///
 /// As with most bare functions, this is not exported by default. You can import
 /// it specifically from `diesel::pg::expression::dsl::any`, or glob import
-/// `diesel::dsl::*`
+/// `diesel::dsl::*`.
 ///
 /// # Example
 ///

